server.js

```diff
@@ -0,0 +1,14 @@
+const express= require("express")
+const server=express();
+//const db= require("./data/dataHelpers.js")
+server.use(express.json());
+server.get('/',async(req,res)=>{
+  res.status(200).json({api:'up'})
+})
+
+module.exports=server
+
+
+
+
+
```

Comment:
Usually you don't want a bunch of whitespace at the end of a file! Preferably, there will be one newline at the end of a file or none.